vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/cbor/CborTextString.kt

```diff
@@ -0,0 +1,8 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList.cbor
+
+import kotlinx.serialization.Serializable
+import kotlin.jvm.JvmInline
+
+@Serializable
+@JvmInline
+value class CborTextString(val value: String)
```

Comment:
This class is never used

Reply:
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b